src/explore-education-statistics-common/src/modules/charts/types/chart.ts

```diff
@@ -91,6 +112,7 @@ export interface ChartProps {
   legend?: LegendConfiguration;
   includeNonNumericData?: boolean;
   showDataLabels?: boolean;
+  map?: MapConfiguration;
```

Comment:
Wouldn't put this on `ChartProps` as it's specific to maps. We could just modify the props for `MapBlockInternal`:

```ts
export interface MapBlockProps extends ChartProps {
  // ...
  map: MapConfiguration;
}
```

Then in `ChartBuilder` we would just need to fix the `chartProps`:

```ts
const chartProps = useMemo<ChartBuilderChartProps | undefined>(() => {
  // other stuff ...
  switch (definition.type) {
    case 'map':
      return {
        ...(baseProps as MapBlockProps),
        map: map ?? {
          dataSetConfigs: [],
        },
        boundaryLevel: options.boundaryLevel ?? 0,
        type: 'map',
      };
  }
}, [...]);
```